hal_st/stm32fxxx/DmaStm.hpp

```diff
@@ -34,9 +34,9 @@ namespace hal
 #else
     struct DmaChannelId
     {
-        constexpr DmaChannelId(uint8_t oneBasedDma, uint8_t oneBasedChannel, uint8_t mux)
+        constexpr DmaChannelId(uint8_t oneBasedDma, uint8_t channelIndex, uint8_t mux)
             : dma{ static_cast<uint8_t>(oneBasedDma - 1) }
-            , channel{ static_cast<uint8_t>(oneBasedChannel - 1) }
+            , channel{ static_cast<uint8_t>(channelIndex) }
```

Comment:
I can't allow this. This change is _quite_ breaking.

Reply:
Sorry, this never meant to make it in the PR